common/trade_goods/00_default.txt

```diff
@@ -441,6 +441,7 @@ vegetables = { #441 territories - ML 439#
 	}
 	country = {
 		slaves_move_cost_modifier = -0.25
+        tribesmen_move_cost_modifier = -0.25
```

Comment:
dear god man, indent! 🤣

Reply:
Strange in my studio code the indent was fine....